### DISCUSSION TITLE
点击节点后，无法切换过去

### DISCUSSION THREAD

**Discussion by GrassBlock1** [libnyanpasu/clash-nyanpasu#620] [Q&A] [answered]
Arch Linux，使用测试版的 appimage 和 AUR 中的 `clash-nyanpasu-bin` 都有此问题。
[屏幕录像_20240316_180901.webm](https://github.com/LibNyanpasu/clash-nyanpasu/assets/46253950/9d195277-8f1f-42bf-a2f3-db7427a9a6a6)

**Comment by greenhat616** [comment, marked as answer]
https://github.com/LibNyanpasu/clash-nyanpasu/issues/553
